model/lifecycle.go

```diff
@@ -709,14 +709,19 @@ func CreateBuildFromVersionNoInsert(creationInfo TaskCreationInfo) (*build.Build
 
 	// create task caches for all of the tasks, and place them into the build
 	tasks := []task.Task{}
+	containsActivatedTask := false
+
 	for _, taskP := range tasksForBuild {
 		if taskP.IsPartOfDisplay() {
 			continue // don't add execution parts of display tasks to the UI cache
 		}
+		if taskP.Activated {
```

Comment:
small thing: can we move this above IsPartOfDisplay (I can't think of a reason we'd need to but just in case)

Also I wonder if it'd make sense to also move the IsGithubCheck into this loop (we can't break but that doesn't matter since we're already not breaking because of this loop)

Reply:
I moved it up. 

And you're right, I don't see why IsGithubCheck can't be part of this. I moved it in.